### PR TITLE
Add nooba.io/backingstore to increase ceph storage

### PR DESCRIPTION
**Pull request by 4n4nd**
Related https://github.com/operate-first/SRE/issues/405

### CODE REVIEW

File: cluster-scope/overlays/prod/moc/smaug/backingstores/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (4n4nd) [reply cluster-scope/overlays/prod/moc/smaug/backingstores/kustomization.yaml]: removed and directly referenced the patch in main kustomization

File: cluster-scope/overlays/prod/moc/smaug/backingstores/backingstore.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: noobaa.io/v1alpha1
```
[the remark this answers is not in the document]
Comment (4n4nd) [reply cluster-scope/overlays/prod/moc/smaug/backingstores/backingstore.yaml]: updated :+1:

**Comment by larsks** [comment]
@4n4nd can you rebase this pr?

**Comment by 4n4nd** [comment]
rebased :+1:

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**